Source/SantaGUI/SNTMessageWindowController.m

```diff
@@ -109,24 +159,24 @@ - (void)loadWindow {
 
 - (IBAction)showWindow:(id)sender {
   [(SNTMessageWindow *)self.window fadeIn:sender];
+  [self addObserver:self forKeyPath:@"self.event.fileBundleHash" options:0 context:NULL];
 }
 
-- (IBAction)closeWindow:(id)sender {
+- (void)closeWindow:(id)sender {
```

Comment:
As this is no longer used from any Cocoa controls (and you're making it no longer an IBAction) we might as well override the `close` method from NSWindowController. Change the method signature to:

`- (void)close`